src/styles/variables.less

```diff
@@ -82,7 +82,7 @@
 @border-standardBorder: 1px solid @color-borderColor;
 @border-lightBorder: 1px solid @color-borderColorLight;
 
-@border-table: 1px solid @color-gray;
+@border-table: 1px solid #767b7f;
```

Comment:
Yeah, it's also [repeated here](https://github.com/appnexus/lucid/pull/231/files#diff-d9e45e4cfffeaed1b27a59a59072ccaaR5)